os-jdg7-launch/added/launch/jboss_modules_system_pkgs.sh

```diff
@@ -0,0 +1,16 @@
+
```

Comment:
There's no need to copy this file if you're using the same thing that's in the eap launch module.  All that is required is adding it to the list of `CONFIGURE_SCRIPTS` in openshift-common.sh